packages/components/tests/integration/components/hds/pagination/nav/index-test.js

```diff
@@ -22,13 +22,23 @@ module('Integration | Component | hds/pagination/nav/index', function (hooks) {
 
   // Test API:
 
-  test('it has the same numbr of pages displayed as are passed in', async function (assert) {
+  test('it has the same number of pages displayed as are passed in for the "numbereed" type', async function (assert) {
     await render(hbs`
       <Hds::Pagination::Nav @totalPages={{10}} @currentPage={{1}} @type="numbered" />
     `);
     assert.dom('.hds-pagination-nav__page-item:nth-child(10)').exists();
   });
 
+  test('it displays a truncated list of page numbers for the "truncated" type', async function (assert) {
+    await render(hbs`
+      <Hds::Pagination::Nav @totalPages={{1000}} @currentPage={{1}} @type="truncated" />
+    `);
+    // displays 7 items (the max number displayed)
+    assert.dom('.hds-pagination-nav__page-item').exists({ count: 7 });
+    // It displays an ellipses for the 5th item
+    assert.dom('.hds-pagination-nav__page-item:nth-child(5)').hasText('...');
+  });
+
```

Comment:
since this is quite tricky algorithm, I would also test that
- when next/prev are clicked, the ellipsis moves
- the ellipsis is in the right position (and the numbers too) for the different combinations
  - `1* 2 3 4 ... 99 100`
  - `1 2* 3 4 ... 99 100`
  - `1 2 3* 4 ... 99 100`
  - `1 ... 3 4* 5 ... 100`
  - `1 ... 96 97* 98 ... 100`
  - `1 ... 97 98* 99 100`
  - `1 ... 97 98 99* 100`
  - `1 ... 97 98 99 100*`

Reply:
I started adding tests for these combinations but ran into trouble. (I commented out the test above.)

It seemed like it wasn't waiting for the DOM to update after the click fired or at least that's my guess. When I try testing it manually within the Scrappy documentation page it seems to work as expected.

Reply:
> It seemed like it wasn't waiting for the DOM to update after the click fired

Can take a look with you next week, but try using `await settled();` and see if that helps things wait

Reply:
I can try it again but it didn't work when I tried before (see comment in code). 

I'll keep experimenting though.

Reply:
After discussing with @didoo earlier today I created some new simplified tests for truncation. I can look into adding a more robust test a little later on if needed (also as discussed).